src/app/styles/theme/ui/breakpoints.js

```diff
@@ -0,0 +1,10 @@
+const breakpoints = {
+  xs: "320px",
+  sm: "360px",
+  md: "768px",
+  lg: "1024px",
+  xl: "1440px",
+  xxl: "1920px",
+};
+
```

Comment:
You would like to add one more utility like:
`export const device = {
  xs: `(min-width: ${size.xs})`,
}`
Or max-width, depends on your approach, mobile first/desktop first, I prefer mobile first, but should be decided within the team as well.

Reply:
Done